src/services/schemas/utilities.ts

```diff
@@ -52,17 +52,17 @@ export function getSchemaRequestValue(schema: Schema, values: SchemaValues): Sch
 /*
  * Gets a UI friendly version of a property's api friendly value. Used for mapping.
  */
-export function getSchemaPropertyResponseValue(property: SchemaProperty, value: SchemaValue): SchemaValue {
-  const service = schemaPropertyServiceFactory(property, 0)
+export function getSchemaPropertyResponseValue(property: SchemaProperty, value: SchemaValue, level?: number): SchemaValue {
+  const service = schemaPropertyServiceFactory(property, level ?? 0)
```

Comment:
```suggestion
export function getSchemaPropertyResponseValue(property: SchemaProperty, value: SchemaValue, level: number = 0): SchemaValue {
  const service = schemaPropertyServiceFactory(property, level)
```

Reply:
For some reason I thought TS didn't respect default values 😮‍💨  nice suggestion